SchoolHousing/SchoolHousing/Models/SchoolHouse.cs

```diff
@@ -1,7 +1,4 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Threading.Tasks;
```

Comment:
Does it still compile and run after removing tehse?  If so, good...  If not, test first and put them back if it doesn't work.